rc/tools/spell.kak

```diff
@@ -49,14 +51,10 @@ define-command -params ..1 -docstring %{
                         # drop the identification message
                     }
 
-                    else if (/^\*/) {
+                    else if (/^[-*+]/) {
```

Comment:
Is `-` a sign produced by Awk that wasn't supported? Why factorise the following scope into this one? Why is this hunk in the commit that only introduces options?

Reply:
Actually as the commit message says it is necessary for languages with affiliation, hunspell use `-` for compounds and `+` for root.